wasmtime-runtime/src/jit_dump.rs

```diff
@@ -0,0 +1,650 @@
+//! Support for jitdump files which can be used by perf for profiling jitted code.
+//! https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/tools/perf/Documentation/jitdump-specification.txt
+//!
+//! Usage Example:
+//!     Record
+//!         sudo perf record -k 1 -e instructions:u target/debug/wasmtime -g -jitdump test.wasm
+//!     Combine
+//!         sudo perf inject -v -j -i perf.data -o perf.jit.data
+//!     Report
+//!         sudo perf report -i perf.jit.data -F+period,srcline
+//! Note: For descriptive results, the WASM file being executed should contain dwarf debug data
+
+use libc::{c_int, c_void, clockid_t, mmap, open, sysconf, timespec};
+use serde::{Deserialize, Serialize};
+use std::ffi::CString;
+use std::fmt::Debug;
+use std::fs::File;
+use std::io::Write;
+use std::{mem, process, borrow};
+use std::os::unix::io::FromRawFd;
+use object::Object;
+use target_lexicon::Architecture;
+
+#[cfg(target_pointer_width = "64")]
+use goblin::elf64 as elf;
+
+#[cfg(target_pointer_width = "32")]
+use goblin::elf32 as elf;
+
+#[repr(C)]
+pub enum RecordId {
+    JitCodeLoad,
```

Comment:
```suggestion
    JitCodeLoad = 0,
```

Same for the other variants to prevent for breaking when someone reorders the variants. If you dont want to add these numbers, please add a comment that order matters for the discriminants.

Reply:
Hi .. thanks a lot for the suggestion. Will make these explicit.